akka-cluster-metrics/src/multi-jvm/scala/akka/cluster/metrics/sample/StatsSampleSpec.scala

```diff
@@ -45,7 +45,7 @@ object StatsSampleSpecConfig extends MultiNodeConfig {
           cluster {
             enabled = on
             allow-local-routees = on
-            use-role = compute
+            use-role-set = ["compute"]
```

Comment:
In many similar we have `with` methods for optional things, `withRoles`.
Here is an example that I worked on the other day, incidentilly also adding support for multi-role: https://github.com/akka/akka/pull/23240/files#diff-2c141d595d12e5b7d90b1931f0cbaf64R130

I guess `ClusterRouterGroupSettings` is changing here. That is even more problematic because it is a `case class`.